### ISSUE TITLE
Model references link to GitLab

### ISSUE THREAD

**Issue by j-a-foster** [ukaea/PROCESS#3124]
Some references at the bottom of model pages in the docs link to issues in the old GitLab repo. I'm not sure if these are publicly accessible and should be updated for more accessibility. The copied over MRs also don't contain the same level of detail as the GitLab ones.

Here's a working list of pages with references that link to GitLab:

- [TF Coil](https://ukaea.github.io/PROCESS/eng-models/tf-coil/)

**Comment by kj5248** [comment]
Also found another reference in standards.md to issue 939 in Gitlab on variable renaming. All of these have been changed in PR alongside links.
